### ISSUE TITLE
.ASS format support

### ISSUE THREAD

**Issue by Spontini** [Stremio/stremio-features#232]
Hello, I think it's needed for the Stremio player, especially for Anime/Cartoon fans .ASS subtitle format is so important

**Comment by aquariumbird** [comment]
I agree

**Comment by shanyan-wcx** [comment]
+1

**Comment by Sparh4wk** [comment]
+1

**Comment by TheAymen** [comment]
Yes, this is a big problem on Android TV. Tried it on phone app and it seems to work smoothly.

**Comment by kKaskak** [comment]
https://github.com/Stremio/stremio-web/pull/1339
https://github.com/Stremio/stremio-video/pull/138 
https://github.com/Stremio/stremio-video/pull/157
https://github.com/Stremio/stremio-video/pull/158
